### PR TITLE
[ADD] add proxy and other lint changes

### PR DESCRIPTION
**Pull request by vibindaniel**

### CODE REVIEW

File: charts/variant-api/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
+---
 apiVersion: v2
 name: variant-api
 description: A Helm chart for APIs to Variant clusters
 
-version: 2.1.31
+version: 2.2.0
```
Comment (vibindaniel) [inline charts/variant-api/Chart.yaml:6]: bumped this

File: .github/workflows/branch-ci.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+
+      - id: lazy-setup
+        uses: variant-inc/actions-setup@v2
+
+      - name: Unit Test
+        shell: pwsh
+        run: ./scripts/unit_test.ps1
+
+      # - name: Lazy Action Octopus
```
Comment (zumic96) [inline .github/workflows/branch-ci.yml:24]: Removing integration tests completely?

File: charts/variant-cron/README.md [hunk 3]
```diff
@@ -13,22 +11,28 @@ A Helm chart for Istio Objects
 ### Prerequisites
 
 1. Use a CloudOps Github CI workflow that publishes an image
-   - [.NET](https://github.com/variant-inc/actions-dotnet)
-   - [Node](https://github.com/variant-inc/actions-nodejs)
-   - [Python](https://github.com/variant-inc/actions-python)
+   - [.NET](<https://github.com/variant-inc/actions-dotnet>)
```
[the remark this answers is not in the document]
Comment (vibindaniel) [reply charts/variant-cron/README.md:14]: I ran a regex replace. It isn't a new way. I'll revert them 😢

File: .github/workflows/branch-ci.yml (hunk 2 above)
Comment (vibindaniel) [reply .github/workflows/branch-ci.yml:24]: yes, we'll find a better way to do it in the future.

File: charts/variant-api/values.yaml [hunk 4]
```diff
@@ -120,29 +121,28 @@ deployment:
...
 awsSecrets: []
   # - name: eng-secret-in-aws
 
-# -- (string) Value for a [label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) named `revision`
+# -- (string) Value for a [label](<https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>) named `revision`
 # that will be applied to all objects created by a specific chart installation.
 # Strongly encouraged that this value corresponds to 1 of: Octopus package version, short-SHA of the commit, Octopus release version
 revision:
 
 # -- (bool) selecting authentication: true when defining an api resource,
```
[the remark this answers is not in the document]
Comment (vibindaniel) [reply charts/variant-api/values.yaml]: cleaned up this and more

File: charts/variant-handler/templates/hpa.yaml
[the remark this answers is not in the document]
Comment (vibindaniel) [reply charts/variant-handler/templates/hpa.yaml:1]: added back

File: charts/variant-handler/values.yaml [hunk 5]
```diff
@@ -118,57 +135,65 @@ deployment:
...
+secretVars:
+  {}
   # foo: bar
 
 # -- (map) User defined environment variables are implemented here.
-# [More Information](<https://backstage.apps.ops-drivevariant.com/docs/default/Component/dx-docs/Apps/Common/environment_variables>)
-configVars: {}
+# [More Information](https://backstage.apps.ops-drivevariant.com/docs/default/Component/dx-docs/Apps/Common/environment_variables)
+configVars:
+  {}
```
Comment (zumic96) [inline charts/variant-handler/values.yaml:161]: Some of these values are changed to have `{}` or `[]` in new line while others are left in one line [example](https://github.com/variant-inc/lazy-helm-charts/pull/115/files/883199392f39cfc4894a5370a9c069647d4670a6..8fa601b8b5537be59ec349beef80246b8701817c#diff-ea7803aba614d13c6738b8d420a9ee0db080b51a198a3d2efb76f42915d0df40R185).

**Review by zumic96 — APPROVED** [review]

**Review by zumic96 — APPROVED** [review]

**Pull request merged**